android/src/main/java/com/stripeterminalreactnative/Mappers.kt

```diff
@@ -888,3 +889,35 @@ fun mapFromCollectDataType(type: String): CollectDataType? {
         else -> null
     }
 }
+
+fun mapToTapZoneIndicator(indicator: String?): LocalMobileUxConfiguration.TapZoneIndicator {
+    return when (indicator) {
+        "default" -> LocalMobileUxConfiguration.TapZoneIndicator.DEFAULT
+        "above" -> LocalMobileUxConfiguration.TapZoneIndicator.ABOVE
+        "below" -> LocalMobileUxConfiguration.TapZoneIndicator.BELOW
+        "front" -> LocalMobileUxConfiguration.TapZoneIndicator.FRONT
+        "behind" -> LocalMobileUxConfiguration.TapZoneIndicator.BEHIND
+        else -> LocalMobileUxConfiguration.TapZoneIndicator.DEFAULT
+    }
+}
+
+fun mapToDarkMode(mode: String?): LocalMobileUxConfiguration.DarkMode {
+    return when (mode) {
+        "dark" -> LocalMobileUxConfiguration.DarkMode.DARK
+        "light" -> LocalMobileUxConfiguration.DarkMode.LIGHT
+        "system" -> LocalMobileUxConfiguration.DarkMode.SYSTEM
+        else -> LocalMobileUxConfiguration.DarkMode.SYSTEM
```

Comment:
I believe the default is actually:

```suggestion
        else -> LocalMobileUxConfiguration.DarkMode.LIGHT
```